Improved the Three Variable Pivot Table dialog

Ivanluv opened the pull request:
Replaces PR#7315 and fixes issues raised in PR #6634 
@rdstern currently the Pivot Table improvements are under the Describe>Three Variable menu. Moving it under the Describe>Two Variable menu bring an error since the file has been manipulated in another PR.
I suggest have this reviewed and possibly merged then I will remove the dialogue under the Describe >Three Variable menu and Move it to the Describe >Two Variable menu

rdstern reviewed (approved):
I am now happy with this dialogue.  Once merged it will change position and replace the pivot-table dialogue on the new Two/Three Variable dialogue.
Over to @shadrackkibet or @lloyddewit

N-thony reviewed (approved):
Looks good

lloyddewit commented on instat/dlgThreeVariablePivotTable.vb:
```diff
@@ -225,6 +242,21 @@ Public Class dlgThreeVariablePivotTable
+    Private Sub ucrChkNumericVariable_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrChkNumericVariable.ControlValueChanged
+        If ucrChkNumericVariable.Checked Then
+            ucrReceiverAdditionalRowFactor.SetMeAsReceiver()
+        ElseIf Not ucrChkNumericVariable.Checked AndAlso ucrChkSelectedVariable.Checked Then
```
`Not ucrChkNumericVariable.Checked` will always be True (see line 254), so we could simplify line 256:
```suggestion
        ElseIf ucrChkSelectedVariable.Checked Then
```

Ivanluv replied:
Done

lloyddewit reviewed (approved):
@Ivanluv Thanks, looks good.
I only had one comment but it's so small that it's not worth delaying approval for

lloyddewit commented:
@Ivanluv In your opening comment for this PR, you state 'fixes issues raised in PR 6634'.
However the link goes to issue #6634. Did you mean the issue or the PR?
Please could you clarify which parts of which issue are fixed by this PR, and close any completed issue(s).
Could we also improve the title of this PR to make it more specific and meaningful?
Thanks

Ivanluv commented:
@lloyddewit sorry for the confusion. This addresses improvements suggested in PR #6634

lloyddewit commented:
> @lloyddewit sorry for the confusion. This addresses improvements suggested in PR #6634

@Ivanluv #6634 is an issue not a PR. Did you mean a different PR number?
P.S. I added a 'TODO' to the top comment, please could you resolve it?

Ivanluv commented:
> > @lloyddewit sorry for the confusion. This addresses improvements suggested in PR #6634
> 
> @Ivanluv #6634 is an issue not a PR. Did you mean a different PR number? P.S. I added a 'TODO' to the top comment, please could you resolve it?

@lloyddewit its issue #6634. I have also fixed the comment
